### ISSUE TITLE
Manual scaling of scalar var appears to do nothing

### ISSUE THREAD

**Issue by lindsayad** [idaholab/moose#17699]
## Bug Description
Applying `scaling` to a scalar variable doesn't appear to scale its residual.

## Steps to Reproduce
At least in an AD sim try toggling on scaling of a scalar variable and monitor its individual variable residual norm. It appears to be unaffected by the value of scaling...and what's worse it appears that the variable's residual then doesn't move during the course of the nonlinear solve!

## Impact
Unable to improve condition number related to poor scaling of global variable.

**Comment by GiudGiud** [comment]
@lindsayad is this fixed by the fix on grouping?

**Comment by lindsayad** [comment]
No. The bug reported here appears to be about manual scaling as opposed to auto scaling

**Comment by GiudGiud** [comment]
oh my bad I read too fast
